roles/network_plugin/multus/templates/multus-daemonset.yml.j2

```diff
@@ -35,13 +35,13 @@ spec:
       containers:
       - name: kube-multus
         image: {{ multus_image_repo }}:{{ multus_image_tag }}
-        command: ["/entrypoint.sh"]
+        command: ["/thin_entrypoint"]
```

Comment:
Hi @ThisIsQasim, thanks for the contributor, I think we should refer to https://github.com/k8snetworkplumbingwg/multus-cni/blob/v4.1.0/deployments/multus-daemonset.yml and see the differences, at least, we need a initContainer to install multus.

Reply:
That's the daemonset config that I used as a reference and it worked without the `initContainer` (it installs the cni binary on main container startup) but you are right it is better to have it.

Reply:
I have added the `initContainer`. I don't see anything else worth adding but please let me know if you find something.

Reply:
Thanks for the update, looks good now!

/lgtm

Reply:
please for the approval, thanks :)

/cc @yankay